api-specifications/properties.json

```diff
@@ -0,0 +1,234 @@
+{
+  "scriptappy": "1.0.0",
+  "info": {
+    "name": "@nebula.js/sn-org-chart:properties",
+    "description": "Org chart generic object definition",
+    "version": "0.15.0",
+    "license": "MIT",
+    "stability": "experimental",
+    "x-qlik-visibility": "public"
+  },
+  "entries": {},
+  "definitions": {
+    "Background": {
+      "kind": "object",
+      "entries": {
+        "colorType": {
+          "description": "colorType - How the font color is defined",
+          "optional": true,
+          "defaultValue": "'colorPicker'",
+          "kind": "union",
+          "items": [
+            {
+              "kind": "literal",
+              "value": "'auto'"
+            },
+            {
+              "kind": "literal",
+              "value": "'colorPicker'"
+            },
+            {
+              "kind": "literal",
+              "value": "'byExpression'"
+            }
+          ],
+          "type": "string"
+        },
+        "color": {
+          "description": "Color defined by index or hex code, needed if colorType is colorPicker",
+          "optional": true,
+          "type": "#/definitions/PaletteColor"
+        },
+        "colorExpression": {
+          "description": "Color defined by expression, needed if colorType is byExpression",
+          "optional": true,
+          "type": "#/definitions/ColorExpression"
+        }
+      }
+    },
+    "Border": {
+      "kind": "object",
+      "entries": {
+        "top": {
+          "description": "top - Set to true to show thicker top border",
+          "optional": true,
+          "defaultValue": true,
+          "type": "boolean"
+        },
+        "fullBorder": {
+          "description": "fullBorder - Set to true to show full border, default is false",
```

Comment:
The description looks odd with the property name there as well. Try changing the jsdoc definition to:
`@property {boolean} [fullBorder=false] Set to true to show full border`